docs/logging-usage.rst

```diff
@@ -250,10 +250,11 @@ Create a Cloud Pub/Sub sink:
 
    >>> from gcloud import logging
    >>> client = logging.Client()
+
    >>> sink = client.sink(
    ...     "robots-pubsub",
-   ...     filter='log:apache-access AND textPayload:robot')
-   >>> sink.pubsub_topic = 'projects/my-project/topics/my-topic'
+   ...      'log:apache-access AND textPayload:robot', 
```

Comment:
Small note, the spacing here seems to be a little different from the other examples.
Also, there's a space at the end of the line as well.